### PR TITLE
feat: Added comparison page for Marketplace & Building Blocks

### PR DESCRIPTION
**Pull request by Jelledb**

### CODE REVIEW

File: docs/marketplace.vs-building-blocks.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+✅ - supported and works well
+
+🟨 - works to some extent
+
+❌ - not supported at all
+
+|                                           | Building Blocks                                                                  | meshMarketplace                                                                                                       |
+|-------------------------------------------|----------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| Provision resources with Terraform        | ✅ This works very easily                                                         | 🟨 Works with [UniPipe Service Broker](https://github.com/meshcloud/unipipe-service-broker), but is more work         |
+| Run Custom Processes outside of Terraform | ❌                                                                                | ✅ The marketplace runs on OSB API which means you can build anything you want as long as it is compliant with OSB API |
```
Comment (sttomm) [inline docs/marketplace.vs-building-blocks.md:21]:
```suggestion
| Run Custom Processes outside of Terraform | ❌                                                                                | ✅ The marketplace runs on OSB API which means you can build anything you want as long as the interface is compliant with OSB API |
```

**Review by sttomm — APPROVED** [review]
lgtm. Only thing we should really change imo is to make clear which of those points, that Building Blocks currently don't support, are planned for the future. I think for getting a full picture it is important to know what to expect from building blocks in future

File: docs/marketplace.vs-building-blocks.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+id: marketplace.vs-building-blocks
+title: Marketplace vs. Building Blocks
+---
+
+[Building Blocks](administration.building-blocks.md) are the newest capability of meshStack to repetitively roll out infrastructure
+by your workspace users. Building Blocks are considered the future of doing so, and this page explains the current gaps, and
+pros and cons of using Building Blocks or the meshMarketplace.
```
Comment (chewbeccy) [inline docs/marketplace.vs-building-blocks.md:8]:
```suggestion
pros and cons of using Building Blocks or existing Service Broker capabilities. In the future Building Blocks will offer further technical implementation types such as Service Brokers.
```

File: docs/marketplace.vs-building-blocks.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+The table below describes each capability and to what extent it is supported by the respective feature (Building Blocks or Marketplace)
+
+✅ - supported and works well
```
Comment (chewbeccy) [inline docs/marketplace.vs-building-blocks.md:12]:
```suggestion
✅ - supported
```

File: docs/marketplace.vs-building-blocks.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+🟨 - works to some extent
+
+❌ - not supported at all
```
Comment (chewbeccy) [inline docs/marketplace.vs-building-blocks.md:16]:
```suggestion
❌ - not supported
```
we should mention in the cell if something is planned for future

File: docs/marketplace.vs-building-blocks.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+|                                           | Building Blocks                                                                  | meshMarketplace                                                                                                       |
+|-------------------------------------------|----------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| Provision resources with Terraform        | ✅ This works very easily                                                         | 🟨 Works with [UniPipe Service Broker](https://github.com/meshcloud/unipipe-service-broker), but is more work         |
```
Comment (chewbeccy) [inline docs/marketplace.vs-building-blocks.md:20]:
```suggestion
| Provision resources with Terraform        | ✅                                                         | 🟨 Works with [UniPipe Service Broker](https://github.com/meshcloud/unipipe-service-broker), but is more work         |
```

File: docs/marketplace.vs-building-blocks.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+| Run Custom Processes outside of Terraform | ❌                                                                                | ✅ The marketplace runs on OSB API which means you can build anything you want as long as it is compliant with OSB API |
+| Automated Deletion of Resource            | ✅ Terraform Building Blocks can run `tf destroy` automatically                   | 🟨 Can work, but requires your own implementation                                                                     |
```
Comment (chewbeccy) [inline docs/marketplace.vs-building-blocks.md:22]:
```suggestion
| Automated Deletion of Resource            | ✅ Terraform Building Blocks can run `tf destroy` automatically                   | 🟨 possible but logic needs to be included in the service broker                                                                    |
```

File: docs/marketplace.vs-building-blocks.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+| Request User Configuration                | ✅ Works, although more variable types (e.g. select) are missing at the moment    | ✅ Works with JSON schema and allows for a lot of different variations                                                 |
```
Comment (chewbeccy) [inline docs/marketplace.vs-building-blocks.md:23]:
```suggestion
| Request User Configuration                | ✅ further variable types such as "select" are planned   | ✅ Works with JSON schema and allows for a lot of different variations                                                 |
```

File: docs/marketplace.vs-building-blocks.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+| Manage Instances via API                  | ❌                                                                                | ✅ But missing deletion of meshServiceInstances                                                                        |
+| Manage Definitions via API                | ❌                                                                                | ❌                                                                                                                     |
+| Custom Pricing Models                     | ❌                                                                                | ✅They can make this a part of the definition                                                                          |
+| Sandbox Development Process               | ❌ When creating a new Building Block, it is directly available to all workspaces | ✅ Developers can create a private service broker that cannot be booked by anyone else                                 |
```
Comment (chewbeccy) [inline docs/marketplace.vs-building-blocks.md:27]: I think we need to clarify what this means

File: docs/marketplace.vs-building-blocks.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+| Allow non-admins to contribute            | ❌ Only Admins can create Building Block definitions                              | ✅ Workspace users can create a service broker and get it approved to be used by everyone else in the meshStack        |
```
Comment (chewbeccy) [inline docs/marketplace.vs-building-blocks.md:28]: what does everyone else mean? An admin or really everyone

File: docs/marketplace.vs-building-blocks.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+| Works without cloud platforms             | ❌ Building Blocks always attach to a cloud tenant                                | ✅ The marketplace works without having a cloud tenant                                                                 |
+| View all consumption in Admin Area        | ✅ There is a list with all building blocks in the Admin Area                     | ❌ There is only a list of service brokers but not service instances                                                   |
+| Tag and Policy Support                    | ❌                                                                                | ❌                                                                                                                     |
+| Integration on Landing Zones              | ✅ Can be set as recommended or mandatory on a landing zone                       | ✅ for tenant-aware services, but ❌ for regular services                                                               |
```
Comment (chewbeccy) [inline docs/marketplace.vs-building-blocks.md:33]:
```suggestion
| Integration on Landing Zones              | ✅ Can be set as recommended or mandatory on a landing zone                       | 🟨  for tenant-aware services                                                             |
```

**Review by chewbeccy — APPROVED** [review]

File: docs/marketplace.vs-building-blocks.md (hunk 8 above)
Comment (Jelledb) [reply docs/marketplace.vs-building-blocks.md:27]: It means that building blocks cannot be made private first for testing purposes. Instead, it is always directly available.

**Pull request merged**